ipyparallel/tests/test_ssh.py

```diff
@@ -14,22 +15,26 @@
 
 
 @pytest.fixture(params=["SSH", "SSHProxy"])
-def ssh_config(ssh_key, request):
+def ssh_config(ssh_dir, request):
+    windows = True if os.name == "nt" else False
```

Comment:
this can be simpler to set a bool from a boolean expression:

```suggestion
    windows = os.name == "nt"
```